arrows/core/initialize_cameras_landmarks.cxx

```diff
@@ -143,6 +143,11 @@ class initialize_cameras_landmarks::priv
   {
   }
 
+  /// Destructor
+  virtual ~priv()
```

Comment:
I don't believe the virtual is needed here since there are no other virtual methods.